apps/src/gamelab/animationListModule.js

```diff
@@ -709,15 +709,17 @@ export function animationSourceUrl(key, props, withVersion = false) {
 
   // 1. If the animation has a sourceUrl it's external (from the library
   //    or some other outside source, not the animation API) - and we may need
-  //    to run it through the media proxy.
-  if (props.sourceUrl) {
-    return assetPrefix.fixPath(props.sourceUrl);
-  }
-
+  //    to run it through the media proxy. (Note - Before 02/2018 -
+  //    uploaded images may/may not have non-null sourceUrls. After 02/2018 -
+  //    uploaded images will have null sourceUrls)
   // 2. Otherwise it's local to this project, and we should use the animation
   //    key to look it up in the animations API.
-  return animationsApi.basePath(key) + '.png' +
-      ((withVersion && props.version) ? '?version=' + props.version : '');
+  let url = (props.sourceUrl) ?
+      assetPrefix.fixPath(props.sourceUrl) : (animationsApi.basePath(key) + '.png');
+
+  // Appending version here to support projects with uploaded images
+  // with sourceUrls.
+  return url + ((props.version) ? '?version=' + props.version : '');
```

Comment:
@epeach @davidsbailey Always appending the version appears to break some projects because the version listed in their manifest does not actually exist on S3.  See [my case study here](https://github.com/code-dot-org/code-dot-org/pull/20751#issuecomment-367040641).

I haven't yet tracked down why this is the case, but as a first step let's not trust the version IDs in the manifest.